sls/Dockerfile

```diff
@@ -2,28 +2,36 @@ FROM alpine:3.9
 
 ENV SERVERLESS_VERSION=1.50.0 \
     NODE_VERSION=10.14.2-r0 \
-    PYTHON3_VERSION=3.6.8-r2 \
     CLOUD_SDK_VERSION=263.0.0 \
     VAULT_VERSION=1.2.0 \
     AWS_CLI_VERSION=1.16.241 \
     CLOUDSDK_CORE_DISABLE_USAGE_REPORTING=true \
-    PATH=$PATH:/usr/local/gcloud/google-cloud-sdk/bin:/usr/local/vault
+    PATH=$PATH:/usr/local/gcloud/google-cloud-sdk/bin:/usr/local/vault:/root/.pyenv/bin:/root/.pyenv/shims
 
 # Install dependencies and cleanup
-RUN apk add --update unzip jq nodejs=$NODE_VERSION nodejs-npm=$NODE_VERSION python3=$PYTHON3_VERSION yarn \
+RUN apk add --no-cache --virtual=.build-deps curl git openssl-dev bzip2-dev zlib-dev readline-dev sqlite-dev \
+  && apk add --no-cache --virutal=.run-deps bash build-base openssl readline libffi libbz2 libffi-dev sqlite sqlite-libs bzip2 zlib unzip jq nodejs=$NODE_VERSION nodejs-npm=$NODE_VERSION yarn \
+  && curl --location https://raw.githubusercontent.com/pyenv/pyenv-installer/master/bin/pyenv-installer | bash \
+  && pyenv install 3.6.8 \
+  && pyenv install 3.7.4 \
+  && pyenv global 3.6.8 3.7.4 \
+  && pyenv rehash \
+  && alias python3=python3.6 \
```

Comment:
You shouldn't need this at all, what are you trying to solve?

Reply:
With alias. `python3` will be an alias for `python3.6` by default, but I wanted to be clear about this, since `python3` will be used in the existing sls projects.